unit_tests/log/test_main_prior_list_serialize.cpp

```diff
@@ -341,6 +341,12 @@ is_active_transaction_server ()
   return true;
 }
 
+bool
+is_page_server ()
+{
+  return false;
+}
+
```

Comment:
```suggestion

bool
is_passive_transaction_server ()
{
  return false;
}
```
if above change is accepted